api/routes/user.py

```diff
@@ -51,3 +51,30 @@ def get_user_profile(
 ):
     """Returns the user profile"""
     return c.get_user_profile(user_uuid, lang, db)
+
+
+@user_router.post("/search-public", status_code=status.HTTP_200_OK, response_model=s.PublicUsersSearchOut)
+def public_search_users(
+    query: s.UserSearchIn,
+    db: Session = Depends(get_db),
+    responses={
+        status.HTTP_409_CONFLICT: {"description": "Selected service not found"},
+    },
+):
+    """Returns filtered list of users"""
+
+    result = c.public_search_users(query, db)
+    return result
+
+
+@user_router.get("/public/{user_uuid}", status_code=status.HTTP_200_OK, response_model=s.PublicUserProfileOut)
+def get_public_user_profile(
+    user_uuid: str,
+    lang: Literal[CFG.UA, CFG.EN] = CFG.UA,  # type: ignore

```

Comment:
Why type ignore?

Reply:
I just copy-pasted it. The error says:

Variable not allowed in type expression PylancereportInvalidTypeForm
Type arguments for "Literal" must be None, a literal value (int, bool, str, or bytes), or an enum value PylancereportInvalidTypeForm

Reply:
This error should no be ignored. May it is a good idea to change it to enum